src/main/resources/application.yml

```diff
@@ -120,6 +120,14 @@ no.nav.security.jwt:
           client-id: ${AZURE_APP_CLIENT_ID}
           client-jwk: ${AZURE_APP_JWK}
           client-auth-method: private_key_jwt
+      dokarkiv-onbehalfof:
+        token-endpoint-url: https://login.microsoftonline.com/${TENANT_ID}/oauth2/v2.0/token
+        grant-type: urn:ietf:params:oauth:grant-type:jwt-bearer
+        scope: api://${DOKARKIV_CLIENT_ID}/.default
```

Comment:
Du snakket i går om å bruke den alternative måten å gjøre det på her?

Reply:
Ja, men jeg fikk verken namespace `org` eller `dokumenthandtering` til å funke, gikk derfor for denne